packages/suite/src/views/wallet/nfts/NftsTable/NftsTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { SelectedAccountLoaded } from '@suite-common/wallet-types';
+import { Card, Column, Table, H3 } from '@trezor/components';
+import { selectNftDefinitions } from '@suite-common/token-definitions';
+
+import { Translation } from 'src/components/suite/Translation';
+import { useSelector } from 'src/hooks/suite';
+import { getNfts } from 'src/utils/wallet/nftUtils';
+
+import NftsRow from './NftsRow';
+import { filterNftTokens } from '@suite-common/wallet-utils';
+import { Token } from '@trezor/blockchain-link-types/src/blockbook-api';
+import { getNetwork } from '@suite-common/wallet-config';
+
+type NftsTableProps = {
+    selectedAccount: SelectedAccountLoaded;
+    searchQuery: string;
+    type: 'ERC721' | 'ERC1155';
+    shown?: boolean;
+    verified?: boolean;
+};
+
+const NftsTable = ({ selectedAccount, searchQuery, type, shown, verified }: NftsTableProps) => {
+    const { account } = selectedAccount;
+    const nftDefinitions = useSelector(state => selectNftDefinitions(state, account.symbol));
+    const filteredTokens = filterNftTokens(account?.tokens || []);
+    const nfts = getNfts({ tokens: filteredTokens, symbol: account.symbol, nftDefinitions });
+    const network = getNetwork(account.symbol);
+
+    const filterNftsByType = (nfts: Token[]) => {
+        return nfts.filter(nft => nft.type === type);
+    };
+
+    const getNftsToShow = () => {
+        if (shown) {
+            return [...nfts.shownVerified, ...nfts.shownUnverified];
+        }
+
+        return verified ? nfts.hiddenVerified : nfts.hiddenUnverified;
+    };
+
+    const nftsToShow = getNftsToShow();
+    const filteredNfts = filterNftsByType(nftsToShow);
+
+    if (filteredNfts.length === 0) {
+        return null;
+    }
+
+    return (
+        <Column width="100%" alignItems="start" gap={12}>
+            <H3>{type}</H3>
+            <Card paddingType="none" overflow="hidden">
+                <Table>
+                    <Table.Header>
+                        <Table.Row>
+                            <Table.Cell colSpan={1}>
+                                <Translation id="TR_COLLECTION" />
+                            </Table.Cell>
+                            {type === 'ERC1155' && (
+                                <>
+                                    <Table.Cell colSpan={1}>
+                                        <Translation id="TR_ID" />
+                                    </Table.Cell>
+
+                                    <Table.Cell colSpan={1}>
+                                        <Translation id="AMOUNT" />
```

Comment:
Quantity would be better imo

Reply:
Amount is used in txs overview, why would it be different here? I think Amount makes sense and it means the same thing as quantity. Also the word quantity has never been used in suite, I would leave it as amount.

Reply:
> Also the word quantity has never been used in suite, I would leave it as amount.
because we did not have NFTs before 😀 

The difference between amount and quantity (how I feel it)
Amount is something divisible, you can have 1.5 ETH, you can have fractions,..
Quantity for me represents discrete, countable units. I have 3x NFT of ID 5. I cant have 2.5 NFT

btw etherscan:
https://etherscan.io/nft/0x177af8c5c96373cb727b8636b331785f49ba24e2/17
![Screenshot 2024-12-03 at 19 47 30](https://github.com/user-attachments/assets/465c35db-25bb-4d3a-9d94-b1bdb865efe7)

Reply:
> Amount is used in txs overview, why would it be different here?

Nobody ever cared about doing NFTs in tx overview nice. Now we start to care, so I think it is not good argument

Reply:
Ok, makes sense